crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -47,9 +51,11 @@ impl ProtocolFee {
         {
             if let Some(partner_fee) = validated_app_data.protocol.partner_fee {
                 let fee_policy = vec![Policy::Volume {
-                    factor: FeeFactor::partner_fee_capped_from(
+                    factor: FeeFactor::try_from_capped(
```

Comment:
Just to double check. The fee policy extracted from the app data here will get stored in the DB when the order gets executed during an auction, right?